lib/index.js

```diff
@@ -1,240 +1,266 @@
+        var type = args[0];
+        var props = args[1];
 
-      if (touchEvents.lastPos.clientX - (touchEvents.lastPos.radiusX || 0) <= box.right &&
-        touchEvents.lastPos.clientX + (touchEvents.lastPos.radiusX || 0) >= box.left &&
-        touchEvents.lastPos.clientY - (touchEvents.lastPos.radiusY || 0) <= box.bottom &&
-        touchEvents.lastPos.clientY + (touchEvents.lastPos.radiusY || 0) >= box.top) {
+        // Check if basic element & has onClick prop
+        if (type && typeof type === 'string' && (
+          (props && typeof props.onClick === 'function') || handleType[type]
+        )) {
+          // Add our own events to props
+          args[1] = propsWithFastclickEvents(type, props || {});
+        }
 
-        if (!isDisabled(event.currentTarget)) {
-          if (typeof onClick === 'function') {
-            copyTouchKeys(touchEvents.lastPos, event);
-            fakeClickEvent(event);
-            onClick(event);
-          }
+        // Apply args to original createElement function
+        return originalCreateElement.apply(null, args);
+      };
 
-          if (!event.defaultPrevented && handleType[type]) {
-            handleType[type](event);
-          }
+      if (typeof React.DOM === 'object') {
+        for (var key in React.DOM) {
+          React.DOM[key] = React.createElement.bind(null, key);
         }
       }
-    }
-  };
-
-  var propsWithFastclickEvents = function (type, props) {
-    var newProps = {};
-
-    // Loop over props
-    for (var key in props) {
-      // Copy props to newProps
-      newProps[key] = props[key];
-    }
-
-    // Apply our wrapped mouse and touch handlers
-    newProps.onClick = onMouseEvent.bind(null, props.onClick);
-    newProps.onMouseDown = onMouseEvent.bind(null, props.onMouseDown);
-    newProps.onMouseMove = onMouseEvent.bind(null, props.onMouseMove);
-    newProps.onMouseUp = onMouseEvent.bind(null, props.onMouseUp);
-    newProps.onTouchStart = onTouchStart.bind(null, props.onTouchStart);
-    newProps.onTouchMove = onTouchMove.bind(null, props.onTouchMove);
-    newProps.onTouchEnd = onTouchEnd.bind(null, props.onTouchEnd, props.onClick, type);
-
-    if (typeof Object.freeze === 'function') {
-      Object.freeze(newProps);
-    }
-
-    return newProps;
+    };
   };
 
-  React.createElement = function () {
-    // Convert arguments to array
-    var args = Array.prototype.slice.call(arguments);
-
-    var type = args[0];
-    var props = args[1];
-
-    // Check if basic element & has onClick prop
-    if (type && typeof type === 'string' && (
-      (props && typeof props.onClick === 'function') || handleType[type]
-    )) {
-      // Add our own events to props
-      args[1] = propsWithFastclickEvents(type, props || {});
+  // Export for commonjs / browserify
+  if (typeof exports === 'object' && typeof module !== 'undefined') {
+    var React = require('react');
+    module.exports = getReactFCInitializer(React);
```

Comment:
In this case, unlike `react-reorder`, since the user is the one doing the intializing you could simple `.bind` the initializer e.g.

```javascript
module.exports = initializeReactFastclick.bind(null, React);
```

But I don't really mind either way. Thoughts?